### DISCUSSION TITLE
Is there a repo for 3d printer recipes?

### DISCUSSION THREAD

**Discussion by KevinTriplett** [markqvist/Reticulum#616] [Help & Questions]
Hi all -- noob here -- I finally jumped in with a LilyGo ESP32 and would like to keep the SMA onboard connector for the antenna. That would mean a change to the [recipe on the website](https://unsigned.io/guides/2023_01_14_Making_A_Handheld_RNode.html). I know there are also other dev boards and so I'm wondering if the community has started a collection of recipes for alternate builds? TIA

**Comment by markqvist** [comment]
There isn't one, as far as I know of, but that would surely be very welcome and useful! Maybe you'd be up for starting such a repo? Would be very nice to have a comprehensive collection of printable files for the various boards that are supported now!

**Comment by KevinTriplett** [comment]
Cool! When I get to the point of having an alternative recipe for the ESP32, I'll reach back out on where to create the repo.
